management-app/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.mvc.view.prefix=/WEB-INF/views/
```

Comment:
consider using `.yml` files instead of `.properties`

Reply:
I will!

Reply:
Done! All config files in both of my modules are now .yml